mysql-test/main/mysql_json_mysql_upgrade_system_tables.test

```diff
@@ -0,0 +1,50 @@
+--echo #
+--echo # MDEV-32462: mysql_upgrade -s still checks for non system tables
+--echo #
+
+# Let's now load plugin first
+-- source include/have_utf8.inc
+--source include/not_embedded.inc
```

Comment:
Please be consistent, either no spaces after `--` or spaces after `--`. I suggest no space after `--` anywhere in the file.